test/unitTests/test-reasoning-layer-lib/test-elements-reasoning-layer.xspec

```diff
@@ -43,7 +43,7 @@
         </x:call>   
         <x:expect label="there is a owl:ObjectProperty" test="boolean(rdf:Description) "/>
         <x:expect label="there is a rdfs:range" test="boolean(rdf:Description/rdfs:range)"/>
-        <x:expect label="there is a correct class URI " test="rdf:Description/rdfs:range/@rdf:resource = 'http://www.w3.org/2004/02/skos/core#Concept'"/>
+        <x:expect label="there is a correct class URI " test="rdf:Description/rdfs:range/@rdf:resource = 'http://data.europa.eu/a4g/ontology#Code'"/>
```

Comment:
we should be expecting `http://www.w3.org/2004/02/skos/core#Concept`.

If it is otherwise, then it is a very old test and needs to be refactored to use skos:Concept.